File: FizzBuzz/Go/roman_kachanovsky/fizz_buzz.go
package roman_kachanovsky

import "strconv"

func FizzBuzz(n int) string {
	if (n % 3) == 0 && (n % 5) == 0 {
		return "Fizz Buzz"
	} else if (n % 3) == 0 {
		return "Fizz"
	} else if (n % 5) == 0 {
		return "Buzz"
	} else {
		return strconv.Itoa(n)
	}
}

File: FizzBuzz/Python/roman_kachanovsky/fizz_buzz.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

from __future__ import unicode_literals


def fizz_buzz(n):
    return 'Fizz' * (not n % 3) + ' ' * (not n % 15) + 'Buzz' * (not n % 5) or str(n)
